src/query/formats/src/file_format_type.rs

```diff
@@ -70,6 +73,7 @@ impl FileFormatOptionsExt {
         settings: &Settings,
     ) -> Result<FileFormatOptionsExt> {
         let timezone = parse_timezone(settings)?;
+        let timestamp_output_format = settings.get_timestamp_output_format()?;
```

Comment:
set to TIMESTAMP_FORMAT here too？

Reply:
Ck handler has already disable, why we contain this func?

Reply:
disabled in settigns by default, can still be used if enabled

Reply:
Done